### DISCUSSION TITLE
ZMK Corne dongle

### DISCUSSION THREAD

**Discussion by alexcamlo** [manna-harbour/miryoku#350] [Q&A] [answered]
I have search everywhere I can think of and found nothing, so before trying to hack it with my small knowledge

Has anyone implemented miryoku + ZMK with dongle for a Corne? (Or any other I could see as an example)

**Comment by manna-harbour** [comment, marked as answer]
@RasmusKoit has been working with dongles recently and has a PR to add support to Miryoku ZMK, which also references an example build: https://github.com/manna-harbour/miryoku_zmk/pull/60
